frontend/package.json

```diff
@@ -32,6 +35,7 @@
   "devDependencies": {
     "eslint": "^8.11.0",
     "eslint-plugin-react": "^7.29.4",
-    "parcel": "^2.3.2"
+    "parcel": "^2.3.2",
+    "process": "^0.11.10"
   }
```

Comment:
I don't think you need to have `process` as a dependency, its a built in node module. I might have missed something though